src/constants/fixed_bases/commit_ivk_r.rs

```diff
@@ -2945,16 +2943,4 @@ mod tests {
         assert_eq!(*coords.x(), pallas::Base::from_repr(GENERATOR.0).unwrap());
         assert_eq!(*coords.y(), pallas::Base::from_repr(GENERATOR.1).unwrap());
     }
-
-    #[test]
-    fn lagrange_coeffs() {
-        let base = super::generator();
-        test_lagrange_coeffs(base, NUM_WINDOWS);
-    }
-
-    #[test]
-    fn z() {
-        let base = super::generator();
-        test_zs_and_us(base, &Z, &U, NUM_WINDOWS);
-    }
```

Comment:
If the helper is re-introduced, re-introduce this test (and likewise for every other constants module).

Reply:
The helper and test has been re-introduced for this and every other constants module.